src/test/java/de/tum/in/www1/artemis/ModelingExerciseIntegrationTest.java

```diff
@@ -264,6 +268,36 @@ public void importModelingExerciseFromCourseToCourse() throws Exception {
         request.postWithResponseBody("/api/modeling-exercises/import/" + modelingExercise.getId(), modelingExercise, ModelingExercise.class, HttpStatus.CREATED);
     }
 
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void importModelingExerciseWithExampleSubmissionFromCourseToCourse() throws Exception {
+        var now = ZonedDateTime.now();
```

Comment:
I think that you can use fixed timestamp instead of now() in tests. The problem is that the created timestamp can’t be controlled by the test. You can’t assert the exact value because it’s always different in every test execution. The rest of the code seems good :)